editor/plugins/skeleton_3d_editor_plugin.cpp

```diff
@@ -655,6 +684,107 @@ void Skeleton3DEditor::_joint_tree_selection_changed() {
 void Skeleton3DEditor::_joint_tree_rmb_select(const Vector2 &p_pos, MouseButton p_button) {
 }
 
+void Skeleton3DEditor::_joint_tree_button_clicked(Object *p_item, int p_column, int p_id, MouseButton p_button) {
+	if (!skeleton) {
+		return;
+	}
+
+	TreeItem *tree_item = Object::cast_to<TreeItem>(p_item);
+	if (tree_item) {
+		String tree_item_metadata = tree_item->get_metadata(0);
+
+		String bone_enabled_property = tree_item_metadata + "/enabled";
+		String bone_parent_property = tree_item_metadata + "/parent";
+		String bone_name_property = tree_item_metadata + "/name";
+		String bone_position_property = tree_item_metadata + "/position";
+		String bone_rotation_property = tree_item_metadata + "/rotation";
```

Comment:
What type of rotation is this? I had a hard time checking it.

Reply:
Seems to be readonly which would avoid the conversion to one of the 6 euler rotations conversion error.

Reply:
In the context of a Skeleton3D, the pose rotation is stored as a Quaternion. The rest is stored as a full Transform3D though.